scico/linop/radon_svmbir.py

```diff
@@ -212,13 +222,10 @@ def __init__(
             W:  Weighting diagonal operator. Must be non-negative.
                 If None, defaults to :class:`.Identity`.
             prox_kwargs: Dictionary of arguments passed to the
-                :meth:`svmbir.recon` prox routine. Note that omitting
-                this dictionary will cause the default dictionary to be
-                used, however omitting entries in the passed dictionary
-                causes the defaults of the underlying :meth:`svmbir.recon`
-                prox routine to be used.
+                :meth:`svmbir.recon` prox routine. Defaults to
+                {"maxiter": 1000, "ctol": 0.001}.
             positivity: Enforce positivity in the prox operation. The
-                loss is not affected.
+                loss is infinite if any element of the input is negative.
```

Comment:
contradicts earlier statement of "unaffected"?

Reply:
Good catch! Fixed it now.